apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js

```diff
@@ -55,6 +68,14 @@ export function getLessonsCompletedByStandardForScript(script) {
   return lessonCompletedByStandard;
 }
 
+export function getNumberLessonsCompleted(script) {
+  return 5;
```

Comment:
where do these numbers come from? should these be dynamically calculated? if not, could they be stored as constants and maybe have a comment explaining why it's always 5 (or 10, for lessons in course)?

Reply:
This is just fake data right now. I was waiting for Erin to finish with the lesson completion logic before actually trying to calculate this.

Reply:
that makes sense! thanks!